### ISSUE TITLE
Is this works for other games?

### ISSUE THREAD

**Issue by mark522412** [michaelkocian/War3Connect3#2]
hi,
thanks for the app but will it work with other games for example Borderlands?
and as for port forwarding im a noob it should be like this right: 
enable DMZ and add local IP and then in port mapping or forwarding add local IP again and the port
is that correct?

**Comment by michaelkocian** [comment]
No, it will not work for any other game. It specifically utilises one feature of the Warcraft 3 protocol.
